topology/metrics.go

```diff
@@ -232,3 +232,190 @@ func (im *InterfaceMetric) Split(cut int64) (common.Metric, common.Metric) {
 
 	return m1, m2
 }
+
+// BlockdevMetric the blockdevice IO stats
```

Comment:
Do you think that this struct could be reused by another probe ? if so we can let it here, if not I would move it in the blockdev folder

Reply:
OK.  I moved it to blockdev.go.  I ran into some problems with the build if I put it in a separate file.